dace/codegen/targets/cpu.py

```diff
@@ -8,6 +8,8 @@
 from dace.codegen.prettycode import CodeIOStream
 from dace.codegen.targets import cpp
 from dace.codegen.targets.common import codeblock_to_cpp
+from dace.codegen.targets.cpp import *
```

Comment:
Avoid `import *`, instead import like `import dace.codegen.targets.cpp as cpp` or similar. This is to avoid polluting the namespace of this file with everything from `cpp`.